services/repository/files/file_test.go

```diff
@@ -10,9 +10,9 @@ import (
 )
 
 func TestCleanUploadFileName(t *testing.T) {
-	assert.Empty(t, CleanGitTreePath(""))
-	assert.Empty(t, CleanGitTreePath("."))
+	assert.Equal(t, "", CleanGitTreePath(""))  //nolint:testifylint // for readability and alignment
```

Comment:
Hmm, adding comments make it harder to read ...


I can help to rewrite the code to something like 

```
for testCase := range cases {
    assert.Equal(....)
}
```

(if you don't mind)

Then we can get rid of the "nolint" tricks

Reply:
Which specific [revive rule](https://revive.run) is being triggered by these underscores? Maybe we can disable the rule itself.

Reply:
I'd vote for disabling `var-naming` completely. This is the kind of rule I don't want to have because it's stylistic.

Reply:
> 2 in `modules/templates/util_json.go` Json -> JSON
> 1 in `modules/templates/util_render.go` Html -> HTML

They would cause large template changes, so I think we shouldn't touch it here.

Reply:
I agree, I hate that naming too, but I think that's just one convention, only enforced by the linter, not enforced by the language. So why do you think we can't disable `var-naming` exactly?

Reply:
You could go onto this branch and `git cherry-pick <hash>` the commit hash from the other branch.